src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Machine;
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Machine> {
+
+    public TruckProducer() {
+    }
+
+    @Override
+    public List<Machine> get() {
```

Comment:
Don't use raw types. Always parametrize interfaces and classes if it is possible. Here, the return type should be `List<Truck>` instead of `List<Machine>`.